CMakeLists.txt

```diff
@@ -111,6 +111,13 @@ if(TC_BUILD_IOS)
   )
   SET(CMAKE_OSX_SYSROOT "${IOS_ISYSROOT}")
   SET(COMPILER_FLAGS "${COMPILER_FLAGS} -Wno-c++11-narrowing -isysroot ${IOS_ISYSROOT}")
+elseif(APPLE)
+  EXECUTE_PROCESS(
+    COMMAND xcrun --sdk macosx --show-sdk-path
+    OUTPUT_VARIABLE MACOS_ISYSROOT
+    OUTPUT_STRIP_TRAILING_WHITESPACE
+  )
+  SET(CMAKE_OSX_SYSROOT "${MACOS_ISYSROOT}")
```

Comment:
[This post](https://stackoverflow.com/a/10170214) suggests that just setting `CMAKE_OSX_SYSROOT` should be sufficient to get correct framework search paths. With this fix, I think the `find_library` commands elsewhere may be able to be simplified to use default paths.